go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/mongodb/mongo-tools
 
-go 1.19
+go 1.20
```

Comment:
I think `minimumGoVersion` in buildscript/build.go also needs to be updated accordingly.